arm/Microsoft.ContainerService/managedClusters/agentPools/readme.md

```diff
@@ -6,23 +6,98 @@ This module deploys an Agent Pool for a Container Service Managed Cluster
+| `scaleSetEvictionPolicy` | string | `Delete` | `[Deallocate, Delete]` | Optional. The eviction policy specifies what to do with the VM when it is evicted. The default is Delete. For more information about eviction see spot VMs |
+| `scaleSetPriority` | string |  | `[Regular, Spot, ]` | Optional. The Virtual Machine Scale Set priority. |
+| `sourceResourceId` | string |  |  | Optional. This is the ARM ID of the source object to be used to create the target object. |
+| `spotMaxPrice` | int | `-1` |  | Optional. Possible values are any decimal value greater than zero or -1 which indicates the willingness to pay any on-demand price. For more details on spot pricing, see spot VMs pricing (https://docs.microsoft.com/en-us/azure/virtual-machines/spot-vms#pricing) |
+| `swapFileSizeMB` | int | `1500` |  | Optional. The size in MB of a swap file that will be created on the node(s). |
+| `tags` | object | `{object}` |  | Optional. Tags of the resource. |
+| `topologyManagerPolicy` | string | `none` | `[none, best-effort, restricted, single-numa-node]` | Optional. Optimize NUMA node-alignment. For more information see Kubernetes Topology Manager (https://kubernetes.io/docs/tasks/administer-cluster/topology-manager) |
+| `transparentHugePageDefrag` | string | `madvise` | `[always, defer, defer+madvise, madvise, never]` | Optional. See Transparent Hugepages (https://www.kernel.org/doc/html/latest/admin-guide/mm/transhuge.html#admin-guide-transhuge) |
+| `transparentHugePageEnabled` | string | `always` | `[always, madvise, never]` | Optional. See Transparent Hugepages (https://www.kernel.org/doc/html/latest/admin-guide/mm/transhuge.html#admin-guide-transhuge) |
+| `type` | string |  |  | Optional. The type of Agent Pool. |
+| `vmMaxMapCount` | int | `65530` |  | Optional. Sysctl setting vm.max_map_count. |
+| `vmSize` | string | `Standard_D2s_v3` |  | Optional. VM size. VM size availability varies by region. If nodes contains insufficient compute resources (memory, cpu, etc) pods might fail to run correctly. For more details on restricted VM sizes, see: /azure/aks/quotas-skus-regions |
+| `vmSwappiness` | int |  |  | Optional. Sysctl setting vm.swappiness. |
+| `vmVfsCachePressure` | int |  |  | Optional. Sysctl setting vm.vfs_cache_pressure. |
+| `vnetSubnetId` | string |  |  | Optional. Node-Subnet ID. If this is not specified, a VNET and subnet will be generated and used. If no podSubnetID is specified, this applies to nodes and pods, otherwise it applies to just nodes. This is of the form: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworks/{virtualNetworkName}/subnets/{subnetName} |
+| `workloadRuntime` | string |  |  | Optional. Determines the type of workload nodes can run. |
 
-### Parameter Usage: `agentPoolProperties`
+### Parameter Usage: `agentPools`
```

Comment:
We don't need this example as it is essentially just the resource itself

Reply:
Removed the example